app/assets/javascripts/modules/moj.AsyncGA.js

```diff
@@ -13,9 +13,9 @@
       window.ga('create', gaTrackingId, 'service.gov.uk');
 
       if (hitTypePage) {
-        window.ga('send', 'page_view', location.pathname + '#' + hitTypePage);
+        window.ga('send', 'pageview', location.pathname + '#' + hitTypePage);
```

Comment:
Can you approve now? Not sure we need a test (can't see this file being looked at again)?